src/main/java/com/partymusicapp/Controllers/SongControllerImpl.java

```diff
@@ -21,12 +21,12 @@ List<Song> getSongList(@PathVariable String partyId){
     }
 
     @DeleteMapping(value = "/{songID}")
-    void RemoveSong(@PathVariable String partyID){
-        songService.RemoveSong(partyID);
+    void RemoveSong(@PathVariable String partyID, String songID){
+        songService.RemoveSong(partyID, songID);
     }
 
     @GetMapping(value = "/partyId", produces = MediaType.APPLICATION_JSON_VALUE)
-    void voteSong(@PathVariable String partyID){
-        songService.voteSong(partyID);
+    void voteSong(@PathVariable String partyID, String songID){
```

Comment:
Aici trebuie sa pui partyId intre {} si sa adaugi in path variabila {songId} si sa ii pui in argumente annotatia @PathVariable, ca altfel nu stie de uunde sa il ia